slang-checks/src/main/java/com/sonarsource/slang/checks/utils/SyntacticEquivalence.java

```diff
@@ -61,4 +66,45 @@ public static boolean areEquivalent(Tree first, Tree second) {
 
     return true;
   }
+
+  public static List<List<Tree>> findDuplicatedGroups(List<Tree> list) {
+    return list.stream()
```

Comment:
Can you please add a unit test for this method?

Reply:
fixed in 189d523